cou/utils/juju_utils.py

```diff
@@ -270,24 +298,20 @@ async def run_action(
         :param action_params: Dictionary of config options for action, defaults to None
         :type action_params: Optional[Dict], optional
         :param raise_on_failure: Raise ActionFailed exception on failure, defaults to False
-        :type raise_on_failure: bool, optional
+        :type raise_on_failure: bool
+        :raises UnitNotFound: _description_
```

Comment:
Missing a simple description
```suggestion
        :raises UnitNotFound: When cannot find a valid unit.
```

Reply:
I saw usage of `_description_` in multiple places, do you know what is actually doing? I was thinking that it's using docstring from exception, but I could not found any information about it.

Reply:
As discussed yesterday, this is automatically generated from a plugin that I use to generate docstrings. We should replace all `_description_` later.